build.js

```diff
@@ -26,5 +26,5 @@ const zip = new AdmZip();
 zip.addLocalFolder(`shared`);
 zip.addLocalFolder(extension);
 
-zip.writeZip(`built/kagi_${extension}_${version}.zip`);
-console.log(`Done: built/kagi_${extension}_${version}.zip`);
\ No newline at end of file
+zip.writeZip(`${__dirname}/built/kagi_${extension}_${version}.zip`);
```

Comment:
I needed this otherwise `npm run build-*` was trying to create a directory inside `built`, which didn't exist. Maybe in the past people have created the directory manually, but it's not in the `README`, and seemed like a bug, since it would effectively put the output in `built/built/` instead of `built/`.